crawler/file_processing.py

```diff
@@ -255,13 +265,15 @@ def __init__(self, file_name: str, centre: Centre):
         # These headers are required in ALL files from ALL lighthouses
         self.required_fields = {
             FIELD_ROOT_SAMPLE_ID,
-            FIELD_VIRAL_PREP_ID,
             FIELD_RNA_ID,
-            FIELD_RNA_PCR_ID,
             FIELD_RESULT,
             FIELD_DATE_TESTED,
         }
 
+        # These are to allow some variability in headers,
+        # due to receiving inconsistent file formats
+        self.header_regex_correction_dict = {"Root Sample": FIELD_ROOT_SAMPLE_ID}
```

Comment:
Should we being this lenient? Shouldn't it only match "Root Sample" or "Root Sample ID"? Something like: `"^Root Sample([ ]?ID) $"`

Reply:
I'm not sure really. I'd err on the side of leaving it more lenient, just because we've received several iterations of the file already, so I wouldn't be surprised if one turns up in future with 'Root Sample Number' or something :)

Reply:
🤔 True, but at the same time, is it possible that they send something unrelated to the "ID" but related to the "Root Sample"? Like "Root Sample Result" or "Root Sample Date Tested"? I hesitant to allow a bug on their side to pollute data on our side but it could be very edge case...

Reply:
Yep OK, that is possible. I'll try to modify it so it allows 'Root Sample' and 'Root Sample ID', and with trailing whitespace or something